streams/src/test/java/org/apache/kafka/streams/integration/LagFetchIntegrationTest.java

```diff
@@ -147,6 +149,9 @@ private void shouldFetchLagsDuringRebalancing(final String optimization) throws
         // create stream threads
         for (int i = 0; i < 2; i++) {
             final Properties props = (Properties) streamsConfiguration.clone();
+            // this test relies on the second instance getting the standby, so we specify
+            // an assignor with this contract.
+            props.put(StreamsConfig.InternalConfig.INTERNAL_TASK_ASSIGNOR_CLASS, PriorTaskAssignor.class.getName());
```

Comment:
This is not a TODO. I'm planning to leave the test like this. (Just opening the floor for objections)

Reply:
It would be nice to take advantage of the now-pluggable assignor and write a test utility assignor that allows you to specify the assignment you want and the validate the inputs that you get. Obviously beyond the scope of the current PR, just a thought I had. Using the PriorTaskAssignor seems like the next best thing so 👍

Reply:
Yep, I had a similar thought, just ran out of motivation after debugging the integration tests.

Reply:
Very understandable